test/hdnode.js

```diff
@@ -180,25 +180,25 @@ describe('HDNode', function () {
       var hd = HDNode.fromBase58(f.master.base58)
       var hdn = hd.neutered()
 
-      assert.equal(hdn.keyPair.d, null)
```

Comment:
This was the only test that broke,  the `keyPair.d` is actually `undefined`, not `null`, in this instance.